website/community/communities.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Communities
+description: The React Native Community
+wrapperClassName: 'community-communities-page'
```

Comment:
```suggestion
```

`wrapperClassName` was required to correct the page appearance, since it was a plain MD page earlier. 

Now, you are using the `content-doc` plugin for the community section pages, which adds the docs pages classes and makes this property redundant.

Reply:
You can also remove `.community-page` references from styles:
https://github.com/facebook/react-native-website/blob/72c8bd5f749ec7d3256af651a7ea1ed3ebd5a9c1/website/src/css/customTheme.scss#L382-L386

Reply:
thanks! done

Reply:
Thank you! 🙏  One more nit, you have left `html[data-theme="dark"].community-page .hash-link` line in styles. 😉